mojo/public/cpp/platform/tcp_platform_handle_utils_posix.cc

```diff
@@ -37,8 +41,6 @@ mojo::PlatformHandle CreateTCPSocket(bool needs_connection, int protocol) {
   return handle;
 }
 
-}  // namespace
-
 int connect_retry(int sockfd, const struct sockaddr* addr, socklen_t alen) {
```

Comment:
It seems that we need to use Capital letter for the first letter of the function name.

Reply:
The underbar is also not necessary. If this is for representing internal helper function only for this file,
then static function would be enough.

Reply:
OK, I'll update.